github-for-jira.sd.yml

```diff
@@ -63,4 +61,27 @@ environmentOverrides:
       remoteDebug: true
     config:
       environmentVariables:
+        WEBHOOK_PROXY_URL: https://github-for-jira.dev.services.atlassian.com/github/events
+        INSTANCE_NAME: github-for-jira-dev
         NODE_OPTIONS: "--no-deprecation --inspect=0.0.0.0:5005"
+        PORT: "8080"
+        # * Commented out env vars are to be saved in secrets
+        # APP_URL: app url for dev instance
+        # WEBHOOK_SECRET: development
+        # ATLASSIAN_URL: Jira instance url
+  staging:
+    config:
+      environmentVariables:
+        APP_URL: INVALID
```

Comment:
btw, should we put proper links here? Or we put "INVALID" to prevent the app from being used in these envs?

Reply:
This was just a placeholder while working on the dev deployment. The staging & prod vars will be updated when we do the deployment work for those stages.